### PR TITLE
Fix:  adds missing text domains to strings.

### PR DESCRIPTION
**Pull request by kebbet**
Solves #393

### CODE REVIEW

**Comment by kebbet** [comment]
This is all the instances I found.

**Comment by hi-hai** [comment]
Thanks for the PR. Only branch `dev` can accept PR. Do you mind to change the branch to `dev` instead of `master`?

**Comment by kebbet** [comment]
Changed with the GUI here on Github, is that ok? – Or should I make a new branch from dev instead and try again?

**Comment by hi-hai** [comment]
This is ok. Thank.

File: src/utility.cls.php [hunk 1]
```diff
@@ -873,8 +873,8 @@ public static function pagination( $total, $limit, $return_offset = false ) {
 		$page_links = paginate_links( array(
 			'base' => add_query_arg( 'pagenum', '%#%' ),
 			'format' => '',
-			'prev_text' => __( '&laquo;', 'text-domain' ),
-			'next_text' => __( '&raquo;', 'text-domain' ),
+			'prev_text' => __( '&laquo;', 'litespeed-cache' ),
```
Comment (hi-hai) [inline src/utility.cls.php:876]: Maybe we should keep this non-translatable.

**Pull request merged**